cfgov/jinja2/v1/_includes/organisms/simple-chart.html

```diff
@@ -40,6 +40,7 @@ <h5>Figure {{ value.figure_number }}</h5>
     <div data-title="{{value.title}}" class="o-simple-chart_selects"></div>
 
     <div class="o-simple-chart_target"
+         data-projected-months='{{ value.projected_months }}'
```

Comment:
Prob should be double quotes, like the surrounding attributes.

Reply:
Nice catch, thanks!